web_app/hazard-reporting-map/components/SocialLogin.jsx

```diff
@@ -0,0 +1,36 @@
+"use client";
+
+import Image from "next/image";
+import Link from "next/link";
+
+const SocialLogins = ({ mode }) => {
+    const handleAuth = (e) => {
+        if (e === "google") {
+            signIn("google", { callbackUrl: "http://localhost:3000/" });
```

Comment:
try not to use hard coded values use env for callbackUrl or use variable instead